docs/source/advanced/mutiny.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Mutiny
+---
+
+Apollo Android includes support for Mutiny.
+
+Apollo types can be converted to Mutiny `Unit` *types* using wrapper functions `MutinyApollo.from(...)` in Java or using extension functions in Kotlin.
```

Comment:
Unit - > Uni ?

Reply:
CTRL + R you know ): the consequences. I'll fix this